.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -0,0 +1,58 @@
+name: Bump versions for extension only
+
+# This workflow covers both '1_check_for_updates' and '2_bump_versions' workflows, therefore it is another variant of the same, but specifically for extension only, without any Prisma CLI updates that would need to be handled.
+
+on:
+    push:
+      branches: 
+        - 'master'
+        - '**.x'
+
+env:
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install Dependencies
+        run: npm install
+      - name: checkout master if necessary
+        if: endsWith(github.ref, '/master') == false
+        run: git checkout master
+      - name: Setup Git
+        run: |
+          git config --global user.email "prismabots@gmail.com"
+          git config --global user.name "Prismo"
+      - name: Update extension version in scripts folder 
+        id: update
+        run: node scripts/bump_extension_version_in_scripts.js dev
+      - name: Push to master
+        run: |
+          git add -A .
+          git commit -m "bump extension version in scripts to ${{ steps.update.outputs.next_extension_version }}."
+          git pull --rebase
+          git push
```

Comment:
This version via file is only tracked in `master` and not the other branches at all?

Reply:
no, because that is not necessary.

Reply:
This might be a question for a future iteration of the scripts, but:
Wouldn't it make sense to have the matching file in the matching branch? (patch-dev in patch-dev and so on?)

Reply:
I don't think this is a good idea, because in order to calculate the next extension version, I need not only the current Prisma Insider extension version for example, but also the currently used Prisma CLI stable version and even the patch-dev version. Moving these to separate branches will be very confusing I think. (also there might not always be a current patch branch)